### PR TITLE
User/chriwall/pipeline build dev home stub

### PR DESCRIPTION
**Pull request by Dreynor**
## Summary of the pull request
Build the Stub Package in the pipeline.

## Detailed description of the pull request / Additional comments
The Stub package is in another Repo: https://microsoft.visualstudio.com/Dart/_git/DevHomeStub?version=GBmain
The pipeline will clone that repo, and build it, then will use the newly created packages in the bundle.
The pipeline changes the version of the manifest for the stub to M.m.p.0 (and the main package is always M.m.p.<1 or HMM>

Changed the name of SDKVersion to VersionOfSDK because msbuild(when building the stub package) has a variable named -SDKVersion, and if the yml file also has one, the yml file overrides the msbuild one, and the namespace "uap" is suddenly unknown, since the xmls part of the AppxManifest file is not known to that SDKVersion.

## Validation steps performed
Ran the pipeline prior to creating PR. Things still built. Will run again in the PR.
Also want to run a "fake release" pipeline run to see the stub get put into the bundle.

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: build/azure-pipelines.yml [hunk 1]
```diff
@@ -127,11 +127,36 @@ stages:
...
+      - task: NuGetToolInstaller@1
+      
+      - task: NuGetAuthenticate@1
+        inputs:
+          nuGetServiceConnections: 'DevHomeInternal'
+      - task: PowerShell@2
+        displayName: Build Stub Packages
+        inputs:
+          filePath: 'Build.ps1'
+          arguments: -Configuration "Release" -Version $(MSIXVersion) -BuildStep "stubpackages"
```
Comment (EricJohnson327) [inline build/azure-pipelines.yml:148]:
```suggestion
          arguments: -Configuration "Release" -Version $(MSIXVersion) -BuildStep "stubpackages" -IsAzurePipelineBuild
```

File: build/azure-pipelines.yml [hunk 2]
```diff
@@ -127,11 +127,36 @@ stages:
         minRiskLevel: 2
         onlyInputs: false
 
+- stage: Build_StubPackages
```
Comment (EricJohnson327) [inline build/azure-pipelines.yml:130]: You'll need to add something similar to .github/workflows/DevHome-CI.yml for PR validation.

Comment (Dreynor) [reply build/azure-pipelines.yml:130]: We decided not to add anything here, and will later add a similar PR validation in the DevHomeStub Repo

**Review by jamespik — APPROVED** [review]

**Pull request merged**